spec/rhc/commands/ssh_spec.rb

```diff
@@ -171,6 +171,26 @@
     end
   end
 
+  describe 'app ssh custom ssh with inaccessible path' do
+    let(:arguments) { ['app', 'ssh', 'app1', '--ssh', '/etc/'] }
+
+    before(:each) do
+      @domain = rest_client.add_domain("mockdomain")
+      @domain.add_application("app1", "mock_type")
```

Comment:
I wonder whether we could put at least the `add_domain` and `add_application` in a `before(:all)` block.